packages/jest-circus/src/event_handler.js

```diff
@@ -35,7 +35,7 @@ const handler: EventHandler = (event, state): void => {
     }
     case 'finish_describe_definition': {
       const {currentDescribeBlock} = state;
-      invariant(currentDescribeBlock, `currentDescribeBlock mest to be there`);
```

Comment:
:D

Reply:
not sure what i was even trying to say :)